src-tests/ConfigHelper.php

```diff
@@ -28,7 +28,7 @@ public static function getDummyConfig()
             'BROWSER_NAME' => 'firefox',
             'ENV' => 'testing',
             'SERVER_URL' => 'http://server.tld:port',
-            'CAPABILITY' => '',
+            'CAPABILITY' => '', // empty
```

Comment:
Well, I want to emphasize this value is intentionally empty, as some tests depends on the emptiness. But OK, this is not so obvious :-).